### DISCUSSION TITLE
Error in making GPU neighboring search for AMD graphic card

### DISCUSSION THREAD

**Discussion by Hassan-Bahrami** [InteractiveComputerGraphics/SPlisHSPlasH#176] [Q&A]
Hi,

I was trying to make the library on my machine with following configuraion:

Windows 10
Cmake 3.19
Visual Studio 2019
AMD Radeon graphic card

When I checked all configurations in the Cmake to make it in my path, I received this error:

"no CUDA toolset found"

After searching in internert, I noticed CUDA only works on NVIDIA graphic cards and as I have AMD, it is not possible to use GPU computation of SPlisHSPlasH on my machine. So, I unchecked GPU neghboring search and made successfully.

I am wondering, is there any way to fix this problem for AMD users so I can use GPU computation of your library? I have heard OpenCL is working on AMD as well, if this can help.

**Comment by janbender** [comment]
Hi, 

OpenCL is a completely different language than Cuda. So there is no simple fix. The only way would be to reimplement everything using OpenCL. So unfortunately there is no quick solution. However, the CPU neighborhood search is also quite fast.
